src/tangd.c

```diff
@@ -26,9 +26,33 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#include <getopt.h>
 
 #include <jose/jose.h>
 #include "keys.h"
+#include "socket.h"
+
+static const struct option long_options[] = {
+	{"port", 1, 0, 'p'},
+	{"listen", 0, 0, 'l'},
+	{"version", 0, 0, 'v'},
+	{"help", 0, 0, 'h'},
+	{NULL, 0, 0, 0}
+};
+
+static void print_help(const char *name)
```

Comment:
Nit: could you please follow the (odd) style of the rest of the file and have it like this:
```
static void
print_help(const char *name)
```

Reply:
done